src/styles/blocks/main/main-rows.scss

```diff
@@ -0,0 +1,20 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 0 73px;
+
+  &__row {
+    display: flex;
+    flex-grow: 1;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 944px;
```

Comment:
2 points:
- where do you get this value from? the checklist, p. 13
- better to limit width for `main`, take a look why
![Screenshot 2023-03-01 at 15 54 59](https://user-images.githubusercontent.com/75496546/222160461-cd188fbd-bafd-41d1-9b35-5615bfa860fe.png)